### ISSUE TITLE
Blurry Memory widget screenshot in "Add widgets" UI

### ISSUE THREAD

**Issue by darkvertex** [microsoft/devhome#3096]
### Dev Home version

0.1401.505.0

### Windows build number

10.0.22631.3593

### Steps to reproduce the bug

Open the "Add widgets" dialog and click through CPU, GPU, then Memory, then Network, and notice how Memory looks damn blurry.

Consider this Memory widget screenshot:
<img width="496" alt="image" src="https://github.com/microsoft/devhome/assets/941331/8b979ff7-79ad-447d-b57a-fe72f0fe5335">

Now compare with the sharpness of the CPU widget screenshot:
<img width="499" alt="image" src="https://github.com/microsoft/devhome/assets/941331/ea19f5e8-3c38-4742-b981-81c5d7e5142e">

This bad screenshot was introduced by PR #2343

### Expected result

I expect all widget screenshots to be the same high-def resolution so they look sharp and not make me question my eyesight or think that antialiasing is on the fritz.

### Actual result

Blurry widget screenshot:
<img width="496" alt="image" src="https://github.com/microsoft/devhome/assets/941331/8b979ff7-79ad-447d-b57a-fe72f0fe5335">

**Comment by krschau** [comment]
@darkvertex Is your screen at a scale other than 100% (Windows Settings -> System -> Display -> Scale)? Interestingly, the smaller resolution fixed the blurriness issue for me. We should figure out how to get it looking best on all resolutions.

**Comment by darkvertex** [comment]
Yes, that screen of mine is set at 150% (as recommended by Windows itself.)

In my case my secondary monitor has a much larger resolution so it makes the sizes of things consistent when dragging one window across them back and forth. At 100% my high DPI screen is uncomfortable.
